config/file_config.go

```diff
@@ -131,6 +132,7 @@ func NewConfig(config, rules string, errorCallback func(error)) (Config, error)
 	c.SetDefault("UpstreamBufferSize", libhoney.DefaultPendingWorkCapacity)
 	c.SetDefault("PeerBufferSize", libhoney.DefaultPendingWorkCapacity)
 	c.SetDefault("MaxAlloc", uint64(0))
+	c.SetDefault("HoneycombLogger.LoggerSamplerThroughput", 5)
```

Comment:
@martin308 @asdvalenzuela this didn't work to set the default. Did we miss something?